### ISSUE TITLE
[Arm64] Tests fail to complete after 10 CPU hours

### ISSUE THREAD

**Issue by sdmaclea** [dotnet/coreclr#11524]
On Arm64/Unix, I am seeing the following tests take an excessive time to complete.

```
JIT/Methodical/tailcall_v4/hijacking/hijacking.sh
JIT/jit64/opt/cse/hugeexpr1/hugeexpr1.sh
JIT/jit64/opt/cse/HugeField1/HugeField1.sh
```

**Comment by sdmaclea** [comment]
cc @dotnet/arm64-contrib

**Comment by jashook** [comment]
Hijacking is a dup of #4122

**Comment by sdmaclea** [comment]
@jashook Looks like different failure symptoms than #4122.

**Comment by jashook** [comment]
I believe the root cause should be the same, just that the test is non-deterministic. Currently I am looking into the failure.

**Comment by sdmaclea** [comment]
I started to take a look at `JIT/jit64/opt/cse/hugeexpr1/hugeexpr1.sh`.  

Observations, the test fails to complete JIT (>7m).  If I `crossgen` the test, the crossgen completes in a reasonable time (1-2minutes).  An the ni version of the test finishes almost instantaneously completes (<1s).  

@dotnet/jit-contrib Recomendations?

**Comment by sdmaclea** [comment]
This appears fixed by #12047.  Closing
